wallet/dapps-communication.md

```diff
@@ -572,6 +572,119 @@ required:
 
 
 
+</tbody>
+</table>
+
+#### Call a Smart Contract
+
+This method is used to call smart contract.
+
+<table>
+<thead>
+<tr>
+<th>Direction</th>
+<th>Type</th>
+<th>Format</th>
+<th>Example</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td>Webpage to extension</td>
+<td><code>account.callSC</code></td>
+<td>
+
+```yaml
+type: object
+properties:
+  nickname:
+    type: string
+    description: Account nickname
+  functionName:
+    type: string
+  at:
+    type: string
+    format: base58check
+    description: Smart contract address
+  args:
+    type: Args
+    description: Smart contract arguments
+  coins:
+    type: string
+    format: BigInt
+    description: Amount of coins to send to the block creator
+  nonPersistentExecution (optional):
+    type: object
+    properties:
+      isNPE:
```

Comment:
If this field (nonPersistentExecution) is optional, what is the default value of this key ?

Reply:
is this a duplicate ?